CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# iwant-bot
+---
+## Hello everyone!
+- :star2: If you are reading this document then I'm assuming that you are interested in contributing to the __iwant-bot__ project.
+- This project is made by Kiwi.com interns. 
+- All contributions are welcome: new use-cases, code, patches, bugfixes, issues, etc. 
+- This is just set of guidelines, not rules. Use your best judgement, and feel free to propose changes in a pull request.
+
+
+## Ways to contribute
+---
+  - At first, check existing [issues](https://github.com/kiwicom/iwant-bot/issues) and our pull requests and help us with development with your proposed solution or improvement
+  - [Create own issue](https://guides.github.com/features/issues/), if you think you have a good idea and let us consider your contribution
+  - To report a bug:
+      - Open an issue with label "bug" :bug:
+      - Describe expected behavior
+      - Describe an actual, incorrect behavior
+      - It would be great if a pull request with possible solution came along with a report
+  - You can create your own [pull request](https://help.github.com/articles/creating-a-pull-request/) on [GitHub](https://github.com/kiwicom/iwant-bot)
+
+### Tools we use for linting, testing and fixing code
+---
+- [coala] - linting and fixing code for all languages
+    - You could [install coala](http://docs.coala.io/en/latest/Users/Install.html) on your computer and check different _bears_ on your local machine
+    - Every commit is automatically tested on [circle CI] with coala (in development)

```

Comment:
it means that coala tests integration is still in a pull request and` (in development)` clause will be removed when #34 will be merged